packages/docs-website/docs/protocol-docs/client-specification/json-rpc-api.md

```diff
@@ -0,0 +1,420 @@
+
+A `Participant` contains identifying information about channel members:
+
+| Name           | Type      | Description                                              |
+| -------------- | --------- | -------------------------------------------------------- |
+| participantId  | string    | Unique identifier of channel participant                 |
+| signingAddress | `Address` | Address used to sign channel state updates               |
+| destination    | `Address` | Address to receive funds in case of disputes/withdrawals |
+
+and an `Allocation` tracks information about the channel balance:
+
+| Name            | Type               | Description                                              |
+| --------------- | ------------------ | -------------------------------------------------------- |
+| token           | `Address`          | Address of asset on chain                                |
+| allocationItems | `AllocationItem[]` | Assigns some asset to a given user by address and amount |
+
+### Parameters
+
+| Name            | Type                            | Description                                |
+| --------------- | ------------------------------- | ------------------------------------------ |
+| participants    | `Participant[]`                 | Identifying information members of channel |
+| allocations     | `Allocation[]`                  | Array of funding amounts for participants  |
+| appDefinition   | string                          | Address of contract governing the channel  |
+| appData         | string                          | Encoded initial state of app               |
+| fundingStrategy | `Direct` \|`Ledger` \|`Virtual` | Funding method (type) of channel           |
+
+### Response
+
+// TODO: (HIGH): What about the `SingleChannelResult` and `MultipleChannelResult` documented in the `server-wallet` package?
+
+This method returns a `ChannelResult` as part of the JSON RPC Response, which is an object with the following fields:
+
+| Name                    | Type              | Description                                |
+| ----------------------- | ----------------- | ------------------------------------------ |
+| participants            | `Participant[]`   | Identifying information members of channel |
+| allocations             | `Allocation[]`    | Array of funding amounts for participants  |
+| appData                 | string            | Encoded initial state of app               |
+| appDefinition           | string            | Address of contract governing the channel  |
+| channelId               | string            | Unique channel identifier                  |
+| status                  | `ChannelStatus`   | Current status of channel                  |
+| turnNum                 | number            | Channel nonce                              |
+| challengeExpirationTime | number (optional) | Time current challenge on channel elapses  |
+
+where the channel status can be one of:
```

Comment:
ref: https://github.com/statechannels/statechannels/issues/2432#issuecomment-669335975

Reply:
I'm still confused as to what the `funding` status is, if there is a better pattern to handle the channel statuses, we should implement it in the finalized schema rather than having irrelevant ones included.

Also the comments mention `these statuses` does that mean opening is also irrelevant or they are just duplicates?

Reply:
I'd comment on the linked issue — this is still open for discussion